Description: kldap_21.08.1.orig.tar.xz
 kldap (21.08.1.1-deepin1) unstable; urgency=medium
 .
   * Fix build with GCC 12
Author: Clansty <i@gao4.pw>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2023-01-06

--- kldap-21.08.1.1.orig/src/widgets/ldapclientsearch.h
+++ kldap-21.08.1.1/src/widgets/ldapclientsearch.h
@@ -131,7 +131,7 @@ public:
      */
     void setAttributes(const QStringList &);
 
-    static Q_REQUIRED_RESULT QStringList defaultAttributes();
+    Q_REQUIRED_RESULT static QStringList defaultAttributes();
 
 Q_SIGNALS:
     /**
